syntax = "proto3";

package robodogs;

message FirebaseMsg {
  
  string json = 1;
}
